checker/tests/interning/Raw3.java

```diff
@@ -54,7 +54,7 @@
   // with raw types
   List<@Interned String> foo6() {
     List<String> sl = new ArrayList<>();
-    //:: warning: [unchecked] unchecked conversion
+    //:: warning: [unchecked] unchecked conversion :: error: (return.type.incompatible)
```

Comment:
We previously wanted that we can always assign a raw type to a generic type - the programmer already get's the javac warning about the unchecked conversion. Are we sure we want to impose this possibly large numbers of errors on people stuck with raw types?